src/wechaty.ts

```diff
@@ -108,12 +108,34 @@ export interface WechatyOptions {
 /**
  * Main bot class.
  *
- * [The World's Shortest ChatBot Code: 6 lines of JavaScript]{@link #wechatyinstance}
+ * A `Bot` is a wechat client depends on which puppet you use.
+ * It may equals
+ * - web-wechat, when you use: [puppet-puppeteer](https://github.com/chatie/wechaty-puppet-puppeteer)/[puppet-wechat4u](https://github.com/chatie/wechaty-puppet-wechat4u)
+ * - ipad-wechat, when you use: [puppet-padchat](https://github.com/lijiarui/wechaty-puppet-padchat)
+ * - ios-wechat, when you use: puppet-ioscat
  *
- * [Wechaty Starter Project]{@link https://github.com/lijiarui/wechaty-getting-started}
- * @example
- * import { Wechaty } from 'wechaty'
+ * See more:
+ * - [What is a Puppet in Wechaty](https://github.com/Chatie/wechaty-getting-started/wiki/FAQ-EN#31-what-is-a-puppet-in-wechaty)
```

Comment:
Please do not use Wiki at `wechaty-getting-started` repository, we have to keep that repo as simple as possible.

I'll disable the wiki of it later, please move the FAQ to the Wiki at Wechaty Repository.

English version should be named with `FAQ`, and the ZH version you can name it.